lib/documents.js

```diff
@@ -2275,18 +2275,17 @@ Documents.prototype.suggest = function suggestDocuments() {
  * @returns {stream.Writable} - a stream.Writable in object mode that receives document descriptor input from the
  *  application for writing to the database.
  */
-Documents.prototype.writeAll = function writeAllDocuments(options) {
+Documents.prototype.writeAll = function writeAllDocuments(stream,options) {
```

Comment:
These all look like breaking changes - technically fine, since this will be in a major release - since a new argument is being added to the beginning of the list of arguments. What's the blast radius of this though? i.e. in addition to mentioning this in release notes, how many changes will be needed to the docs to ensure they don't show any examples from the previous way of invoking these methods?

Reply:
The radius is big. I will work on the functional tests and examples next and also with the doc team to update the documentation.